src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -608,7 +608,17 @@ public BoundCall Call(BoundExpression receiver, MethodSymbol method, ImmutableAr
             return new BoundCall(
                 Syntax, receiver, method, args,
                 ImmutableArray<String>.Empty, ImmutableArray<RefKind>.Empty, false, false, false,
-                ImmutableArray<int>.Empty, LookupResultKind.Viable, method.ReturnType)
+                default(ImmutableArray<int>), LookupResultKind.Viable, method.ReturnType)
+            { WasCompilerGenerated = true };
+        }
+
+        public BoundCall Call(BoundExpression receiver, MethodSymbol method, ImmutableArray<RefKind> argumentRefKinds, ImmutableArray<BoundExpression> args)
```

Comment:
Apparently whatever need I had of it evaporated, because it was unreferenced. I will delete it.